fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -460,6 +461,10 @@
     @VariableMgr.VarAttr(name = ENABLE_COLUMN_EXPR_PREDICATE)
     private boolean enableColumnExprPredicate = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_REWRITE_ZONE_MAP_PREDICATE)
```

Comment:
I think would better directly remove this variable if the default value is true, there are too many codes for this flag.